modules/environment-roles/root.tf

```diff
@@ -98,7 +98,7 @@ data "template_file" "keycloak_terraform_policy_a" {
 
 resource "aws_iam_policy" "keycloak_terraform_iam_a" {
   policy = data.template_file.keycloak_terraform_policy_a.rendered
-  name   = "keycloak-terraform-${local.environment}-a"
+  name   = "TDRKeycloakTerraform${title(local.environment)}-a"
```

Comment:
What are the `-a` and `-b` versions for? I'm struggling to figure it out from the Terraform files.

Reply:
There is a size limit on the size of policies so to enable to add all the necessary permissions to allow Terraform to create resources needed to split the policies out.

Reply:
I see, thanks!

Maybe they could be called "part-a" or "part1" or something? Then it will be more obvious that they're continuations of the same thing, rather than alternatives, which is how I originally interpreted it.

An explanation in the README or the Terraform script would be useful too.

Reply:
Resolved